Rakefile

```diff
@@ -77,19 +88,29 @@ task :lint do
   sh "npm run-script lint"
 end
 
-desc "Run webpack with development settings"
-task :webpack do
-  sh "BUILD_TYPE=dev npm run-script webpack"
+desc "Run webpack with development features turned on. Build is optimized."
```

Comment:
Remove `Build is optimized.`, as there is a conditional in this task to build whether or not `enable_optimizations` is set.